Library/Homebrew/cmd/audit.rb

```diff
@@ -193,6 +194,9 @@ def audit_file
     if present.include?("head") && present.include?("head block")
       problem "Should not have both `head` and `head do`"
     end
+    if present.include?("bottle modifier") && present.include?("bottle block")
```

Comment:
I left this be for now, but I'm happy to go make changes in this PR if there's consensus on the path to take.